packages/contracts-bedrock/scripts/Deploy.s.sol

```diff
@@ -272,9 +283,10 @@ contract Deploy is Deployer {
 
     /// @notice Internal function containing the deploy logic.
     function _run() internal {
-        console.log("start of L1 Deploy!");
-        deploySafe();
-        console.log("deployed Safe!");
+        deploySystemOwnerSafe();
+        console.log("deployed System Owner Safe!");
+        deploySecurityCouncilSafe();
+        console.log("deployed Security Council Safe!");
```

Comment:
Does this imply that using the SC approach we now have is a requirement to use this deploy script? I think that's ok for now since we really need to improve our deploy processes anyway